packages/export/package.json

```diff
@@ -34,5 +34,8 @@
     "lodash": "^4.17.21",
     "minimal-csv-formatter": "^1.0.15"
   },
-  "packageManager": "yarn@3.3.1"
+  "packageManager": "yarn@3.3.1",
+  "peerDependencies": {
```

Comment:
This is a regular dep, not a peer dep. Peer deps won't be installed automatically and you need this to run exports so it's not optional.